invoker/conformance/buildpack_pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud.functions.invoker</groupId>
+  <artifactId>conformance</artifactId>
+  <version>1.1.1-SNAPSHOT</version>
```

Comment:
Will `prerun.sh`/the function build still work if this is `1.1.1-SNAPSHOT` but the actual FF repo moves onto a future version like `1.1.5-SNAPSHOT`? You may have to muck around with the actual FF's pom.xml to force a deterministic number.

I had to do some minor hackery in node to temporarily change the FF node to use a fake, deterministic version before I installed it: https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/test/conformance/prerun.sh#L15

Reply:
Good point. Using a fake version now for invoker and functions-framework-api.